src/main/java/com/fundation/webservice/controller/Controller.java

```diff
@@ -864,7 +876,7 @@ else if (configJson.getString("metadata").equals("xmp")) {
      * Endpoint for downloading zip file containing the products of a conversion process.
      *
      * @param fileName Name of the output zip file
```

Comment:
There's a missing point at  the end of the line.

Reply:
I fix that.